mos-platform/common/lib/main.S

```diff
@@ -20,5 +20,6 @@ __exit:
    LDX mos8(__rc2)
    RTS
 
+.data
```

Comment:
Instead, use .noinit; this saves it from being included in the binary or memset to zero on init.